apps/web/components/Cookiebar/Cookiebar.vue

```diff
@@ -111,7 +111,7 @@
       <div class="w-full flex flex-col xl:flex-row mt-5 gap-2 mb-2">
         <div class="flex-1">
           <UiButton
-            class="w-full h-12"
+            class="w-full h-10"
```

Comment:
Could you please confirm that you've checked for PSI errors using both plugins, as outlined in the task requirements ?

<img width="1217" alt="Screenshot 2024-10-01 at 15 31 59" src="https://github.com/user-attachments/assets/bf501457-f1aa-433a-a2bf-6dca0cc3c543">